fw/ctrl_832/mmc.c

```diff
@@ -174,16 +164,9 @@ unsigned char MMC_Init(void)
 
                 DisableCard();
 
-                // set appropriate SPI speed
-//                if (GetSPIMode() == SPIMODE_FAST)
-//                    AT91C_SPI_CSR[0] = AT91C_SPI_CPOL | (3 << 8); // 16 MHz SPI clock (max 20 MHz for MMC card)
```

Comment:
This code was commented out anyway, the new code removes the commented out code and adds a getcapacity. Nothing here would conflict.